polyfills/Array/from/tests.js

```diff
@@ -49,42 +49,49 @@ describe('returns an array with', function () {
 		var map;
 		var mapIterator;
 
-		if ('Map' in window && 'entries' in Map.prototype) {
+		function returnArgs(){
+			return Array.prototype.slice.call(arguments);
+		}
 
+		if ('Map' in window) {
 			map = new Map();
 			map.set(1,2);
 			map.set(3,4);
-			mapIterator = map.values();
-
-			if (typeof mapIterator.next === 'function') {
+			if (map.values) {
+				mapIterator = map.values();
 
-				// Test map iterable
-				proclaim.deepEqual(Array.from(mapIterator), [2,4]);
+				if (typeof mapIterator.next === 'function') {
 
-				it('can convert from Map', function () {
-					proclaim.deepEqual(Array.from(map), [[1,2],[3,4]]);
-				});
+					// Test map iterable
+					proclaim.deepEqual(Array.from(mapIterator), [2,4]);
```

Comment:
The map and the map iterator are both tested - this is simply the way the dif makes it look. Prior to the tests (and this is exactly the same for sets) the test only tested using the 'map' variable if the Map.prototype contained a 'entries' method (although it erroneously actually used the 'values' method i.e. NOT the method whos existance was tested for). 

Before changes to tests:

	if ('Map' in window && 'entries' in Map.prototype) {

		map = new Map();
		map.set(1,2);
		map.set(3,4);
		mapIterator = map.values();

		if (typeof mapIterator.next === 'function') {

			// Test map iterable
			proclaim.deepEqual(Array.from(mapIterator), [2,4]);

			it('can convert from Map', function () {
				proclaim.deepEqual(Array.from(map), [[1,2],[3,4]]);
			});
		}
	}

after changes to test:

	if ('Map' in window) {
		map = new Map();
		map.set(1,2);
		map.set(3,4);
		if (map.values) {
			mapIterator = map.values();

			if (typeof mapIterator.next === 'function') {

				// Test map iterable
				proclaim.deepEqual(Array.from(mapIterator), [2,4]);
			}
		}
		
		it('can convert from Map', function () {
			proclaim.deepEqual(Array.from(map), [[1,2],[3,4]]);
			proclaim.deepEqual(Array.from(map, returnArgs), [[[1,2],0],[[3,4],1]]);
		});
	}